lesson_19_web_hm/web.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask, render_template, request
+from app import Intake, engine
+from sqlalchemy.orm import Session
+
+
+app = Flask(__name__)
+
+
+@app.route('/',  methods=['GET', 'POST'])
+def user_intake():
+    if request.method == 'POST':
+        user_name = request.form['user']
+        food = request.form['food']
+        weight = request.form['weight(grams)']
+        proteins = request.form['proteins']
+        fats = request.form['fats']
+        carbohydrates = request.form['carbohydrates']
+        intake = {
+            "user_name": user_name,
+            "food": food,
+            "weight": weight,
+            "proteins": proteins,
+            "fats": fats,
+            "carbohydrates": carbohydrates
+        }
+        print(intake)
+        with Session(engine) as db:
+            intake = Intake(user_name=user_name, food=food, weight=weight, proteins=proteins, fats=fats,
```

Comment:
I think you can just pass `request.form` to `Intake` class. Like:
`Intake(**request.form)`